sky/data/data_utils.py

```diff
@@ -589,73 +591,144 @@ def get_cos_regions() -> List[str]:
     ]
 
 
-class Rclone():
-    """Static class implementing common utilities of rclone without rclone sdk.
-
-    Storage providers supported by rclone are required to:
-    - list their rclone profile prefix in RcloneClouds
-    - implement configuration in get_rclone_config()
-    """
-
-    RCLONE_CONFIG_PATH = '~/.config/rclone/rclone.conf'
-    _RCLONE_ABS_CONFIG_PATH = os.path.expanduser(RCLONE_CONFIG_PATH)
-
-    # Mapping of storage providers using rclone
-    # to their respective profile prefix
-    class RcloneClouds(Enum):
-        IBM = 'sky-ibm-'
+class Rclone:
+    """Provides methods to manage and generate Rclone configuration profile."""
+
+    class RcloneStores(enum.Enum):
```

Comment:
I think we should try to get rid of this enum and move the relevant functions to the AbstractStore subclasses. We don't need to do this in this PR, but we can add a TODO comment.